tests/helpers/perf_run_helper.rb

```diff
@@ -29,6 +35,14 @@ module PerfRunHelper
   METRIC_RESULTS_DIR = "/root/gatling-puppet-load-test/simulation-runner/results"
   CURRENT_TUNE_SETTINGS = "current_tune_settings.json"
 
+  def logger
```

Comment:
Explain why this is here!
Two contexts:
* Run with beaker (uses inherited beaker logger)
* Run without beaker (creates logger)